pyproject.toml

```diff
@@ -80,6 +80,7 @@ dependencies = [
     "ipykernel==6.23.1",
     "plotly==5.14.1",
     "pydantic==1.*",
+    "skellytracker==2023.12.1006",
```

Comment:
@jonmatthis do we want to be pinning these subrepos to precise versions (i.e. with `==` rather than `>=`)? In theory we're in control of any breaking changes.